docs/troubleshooting.md

```diff
@@ -0,0 +1,10 @@
+## Conflicting Package Names
+
+If two Operator Sources claim the same package name the Marketplace operator does not allow the user to pick which grpc pod the operator is installed from. Instead, the Marketplace operator will randomly select one of the existing grpc pods to deploy the operator from. To avoid unexpected behavior users should avoid adding any operators with conflicting package names.
+
+Users can view packages available within their current namespace with the following command:
+```bash
+oc get packagemanifests
+```
+
+The operator-sources that are shipped with openshift will never have conflicting package names with one another.
```

Comment:
This is also a detail that is probably not necessary in open source documentation.

Reply:
Removed.